stream.go

```diff
@@ -155,6 +155,15 @@ func (cs *clientStream) Trailer() metadata.MD {
 }
 
 func (cs *clientStream) SendMsg(m interface{}) (err error) {
+	if EnableTracing {
+		cs.traceInfo.tr.LazyLog(payload{m}, true)
```

Comment:
It holds 10 events, and drops from the middle, so it'll be something like 7-8 payloads (excluding the first line, the last line, and one that says how many events were dropped). Yes, it's definitely bounded in that regard.

Nothing bounds the payload here. It'll just stringify it. We need to improve how that's handled, but it's not fatal since an active server will only retain a fixed number of traces in total, so it'll still be bounded in some sense.